pkg/common/malloc/default.go

```diff
@@ -17,9 +17,28 @@ package malloc
 import (
 	"os"
 	"runtime"
+	"strconv"
 	"strings"
+
+	"github.com/matrixorigin/matrixone/pkg/logutil"
+	"go.uber.org/zap"
 )
 
+// checkFraction controls the fraction of checked deallocations
+// On average, 1 / fraction of deallocations will be checked for double free or missing free
+var checkFraction = func() uint32 {
+	// default fraction, large enough to not effecting performance too much
```

Comment:
default should be 0, that is, no debug alloc.

Reply:
done